packaging/samba.spec.j2

```diff
@@ -575,10 +575,12 @@ Requires: %{name} = %{samba_depver}
 Requires: %{name}-client-libs = %{samba_depver}
 Requires: %{name}-common-libs = %{samba_depver}
 Requires: %{name}-libs = %{samba_depver}
+%if 0
```

Comment:
Why `%if 0`?

Reply:
Do you mean why `%if 0` instead of removing it?
I have done it in other locations as well.
At some point we might want to reconcile with the other spec files that have `%if with_internal_tdb` or so...

Reply:
> Do you mean why `%if 0` instead of removing it?

Yes

> I have done it in other locations as well.
> At some point we might want to reconcile with the other spec files that have `%if with_internal_tdb` or so...

Ok.